boards/shields/xenvm_dom0/boards/qemu_cortex_a53.overlay

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &sram0;
+/delete-node/ &pcie;
+/delete-node/ &uart0;
+
+/ {
+	chosen {
+		/delete-property/ zephyr,flash;
+		/delete-property/ zephyr,pcie-controller;
+	};
+
+	/*
+	 * This node may differs on different setups, please check
+	 * following line in Xen boot log to set it right:
+	 * (XEN) Grant table range: 0x00000040200000-0x00000040240000
+	 */
+	hypervisor: hypervisor@40200000 {
+		compatible = "xen,xen";
+		reg = <0x0 0x40200000 0x0 0x40000>;
+		interrupts = <GIC_PPI 0x0 IRQ_TYPE_EDGE IRQ_DEFAULT_PRIORITY>;
+		interrupt-parent = <&gic>;
+		status = "okay";
+	};
+
+	/*
+	 * This node may differs on different setups, because Xen picks
+	 * region for Domain-0 for every specific configuration. You can
+	 * start Xen for your platform and check following log:
+	 * (XEN) Allocating 1:1 mappings for dom0:
+	 * (XEN) BANK[0] 0x00000060000000-0x00000080000000
```

Comment:
Please expand this comment by describing that Xen passes actual memory ranges in device tree, but Zephyr is not capable of parsing device tree in run-time.

Reply:
Fixed